admin_app/src/app/content/edit/page.tsx

```diff
@@ -69,7 +83,21 @@ const AddEditContentPage = () => {
           sx={{ px: sizes.doubleBaseGap, mx: sizes.smallGap }}
         >
           <Layout.Spacer multiplier={2} />
-          <ContentBox content={content} setContent={setContent} />
+          <ContentBox
+            contentId={content_id!}
+            content={content}
+            setContent={setContent}
+            contentData={contentData}
+            setContentData={setContentData}
+            languageId={language_id!}
+            onSaveSuccess={handleSaveSuccess} />
+          <SavedContentDialog
```

Comment:
I thought about that but the back button will not necessarily sending to the content page. Since we have multilingual data, it redirects to the other languages versions instead. So the best way would be to add a close button. What do you think ?